homeassistant/components/tplink/entity.py

```diff
@@ -265,6 +265,7 @@ def _description_for_feature[_D: EntityDescription](
         descriptions: Mapping[str, _D],
         *,
         child_alias: str | None = None,
+        **kwargs: Any,
```

Comment:
Can we subclass this function so the per platform args are passed typed before it gets down here?

Reply:
The way this was handled in the sensor platform was by setting those extra attributes directly on the entity in `async_update_attributes` rather than in the description (.i.e `native_unit_of_measurement`). Can you not do something similar here with the native min and max values?

Reply:
Modified to set the values in the update callback, as they are dynamic and may change (e.g., temperature offset limits are iirc configurable and may differ from celsius to fahrenheit.